modules/issue_tracker/templates/menu_my_issue_tracker.tpl

```diff
@@ -0,0 +1,212 @@
+                            </label>
+                            <div class="col-sm-12 col-md-8">
+                                {$form.maxDate.html}
+                            </div>
+                        </div>
+                        <div class="form-group col-sm-4">
+                            <label class="col-sm-12 col-md-4">
+                                {*{$form.watching.label}*}
+                                &nbsp;
+                            </label>
+                            <div class="col-sm-12 col-md-8">
+                                <b>{$form.watching.html}</b>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="'row">
+                        <hr>
+                    </div>
+                    <div class="row">
+                        <div class="form-group col-sm-4">
+                            <label class="col-sm-12 col-md-4">
+                                {$form.PSCID.label}
+                            </label>
+                            <div class="col-sm-12 col-md-8">
+                                {$form.PSCID.html}
+                            </div>
+                        </div>
+                        <div class="form-group col-sm-4">
+                            <label class="col-sm-12 col-md-4">
+                                {$form.visitLabel.label}
+                            </label>
+                            <div class="col-sm-12 col-md-8">
+                                {$form.visitLabel.html}
+                            </div>
+                        </div>
+                    </div>
+                    <br>
+                    <div class="row">
+                        <div class="form-group col-sm-4 col-md-2 col-xs-12">
+                            <input type="button"
+                                   name="newIssue"
+                                   value="Add Issue"
+                                   class="btn btn-sm btn-primary col-xs-12"
+                                   onclick="location.href='{$baseurl}/issue_tracker/edit/?issueID=0&backURL=/issue_tracker/&subtest=newIssue'"/></div>
+                        <div class="form-group col-sm-4 col-md-2 col-xs-12 col-sm-offset-2 col-md-offset-5">
+                            <input type="submit"
+                                   class="btn btn-sm btn-primary col-xs-12"
+                                   name="filter"
+                                   value="Show Data"></div>
+                        <div class="form-group col-sm-4 col-md-2 col-xs-12">
+                            <input type="button"
+                                   name="reset"
+                                   value="Clear Form"
+                                   class="btn btn-sm btn-primary col-xs-12"
+                                   onclick="location.href='{$baseurl}/issue_tracker/?reset=true'"/></div>
```

Comment:
"Clear Form" button resets to the All Issues tab, should stay on My Issues tab instead -- e.g. add to URL: 
submenu=my_issue_tracker/